__tests__/core/api.test.ts

```diff
@@ -156,7 +156,7 @@ test("all methods mentioned in API docs", () => {
 })
 
 test("only accepted dependencies", () => {
-    const validDeps: string[] = []
+    const validDeps: string[] = ["ts-essentials"]
```

Comment:
😢 I made us have a dependency

Reply:
Womp womp

We'll figure it out! I think it's a reasonable one for now, and we can look into removing it later. Let's ship a patch to fix this in the interim, and I'll put together a follow-up to investigate removing it.